pallets/capacity/src/tests/rewards_provider_tests.rs

```diff
@@ -1,16 +1,16 @@
 use super::mock::*;
 use crate::{
-	Config, CurrentEraInfo, Error, ProviderBoostHistories, ProviderBoostHistory,
-	ProviderBoostRewardsProvider, RewardEraInfo, StakingDetails, StakingType::*,
-	UnclaimedRewardInfo,
+	Config, ProviderBoostHistories, ProviderBoostHistory, ProviderBoostRewardsProvider,
+	StakingType::*, UnclaimedRewardInfo,
 };
-use frame_support::{assert_err, assert_ok, traits::Len};
+use frame_support::{assert_ok, traits::Len};
+use frame_system::pallet_prelude::BlockNumberFor;
```

Comment:
I get a warning here about an unused import, just fyi

Reply:
Also found and fixed an intermittently failing test